.github/workflows/deploy-sdk.yml

```diff
@@ -119,16 +127,26 @@ jobs:
           SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
   
   deploy-cocoapods:
-    needs: [deploy-git-tag] # run after git tag is made 
-    if: needs.deploy-git-tag.outputs.new_release_published == 'true' # only run if a git tag was made.
-    env:    
-      COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
     name: Deploy SDK to Cocoapods 
+    needs: [deploy-git-tag] # run after git tag is made 
```

Comment:
I am still a little confused, 

We are still dependent on `deploy-git-tag` and it would only run 

```
if: ${{ github.event_name == 'push' }} # only deploy tag when triggered automatically by merging a PR.
```

so how will this work?

Reply:
I have not tested this myself, but I believe that the workflow should still work. [The docs for `needs`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds) suggests that because we added an `if` to deploy cocoapods job, this should not be a problem.